### PR TITLE
Maybe fixes cyborg name bug

### LINKED ISSUE #22844: borgs can have no name
states, "Ow."
http://i.imgur.com/1BfXX4J.png

### PR DESCRIPTION
**Pull request by coiax**
It might fix #22844, but I can't replicate the bug locally, which makes
testing difficult.

### CODE REVIEW

**Comment by Armhulen** [comment]
thanks

**Comment by SweaterKittens** [comment]
@coiax If you'd like to test it on the main server one of these nights while I'm on, I'd be happy to spawn you whatever you need to try and replicate it.

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 1]
```diff
@@ -213,7 +213,7 @@
 		changed_name = custom_name
 	if(changed_name == "" && client)
 		changed_name = client.prefs.custom_names["cyborg"]
-	if(changed_name == "")
+	if(!changed_name || changed_name == "")
```
Comment (RemieRichards) [inline code/modules/mob/living/silicon/robot/robot.dm]: ```""``` is false, so now the second half of the if is redundant.

Comment (Jordie0608) [inline code/modules/mob/living/silicon/robot/robot.dm]: Isn't this invalidated by the L214 anyway? I don't recall if the config name for borg would be blank by default.

Comment (coiax) [inline code/modules/mob/living/silicon/robot/robot.dm]: @MrStonedOne tells me that old client prefs can have null for the cyborg name. So I think this is where the bug comes from.

Comment (RemieRichards) [inline code/modules/mob/living/silicon/robot/robot.dm]: does not change my comment, only jordie's.

**Pull request merged**